samples/openthread/cli/CMakeLists.txt

```diff
@@ -6,12 +6,17 @@
 cmake_minimum_required(VERSION 3.13.1)
 
 set(NRF_SUPPORTED_BOARDS
-	nrf52840dk_nrf52840
-	nrf52833dk_nrf52833
+  nrf52840dk_nrf52840
+  nrf52833dk_nrf52833
 )
 
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
-
+include(${NRFXLIB_DIR}/common.cmake)
```

Comment:
is this needed here or by the common.cmake? If by common.cmake maybe move it there?

Reply:
done